dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -91,4 +91,4 @@ console.format.simple=Use the simple logging format.
 console.format.json=Use the JSON logging format.
 
 json.fields=JSON field names that are being replaced
-json.fields.desc=When logs are in JSON format, use this attribute to replace default field names with new field names. Configure the new field name by using the following format: defaultFieldName:newFieldName. For field names that are associated with logs of a specified source, use the following format: [source:]?defaultFieldName:newFieldName, where [source] is the source you want to specify (such as message, trace, or accessLog). To rename multiple fields, specify a comma-separated list of field name mappings.
\ No newline at end of file
+json.fields.desc=When logs are in JSON format, use this attribute to replace default field names with new field names. Use this attribute to omit fields as well. Configure the new field name by using the following format: defaultFieldName:newFieldName?. For field names that are associated with logs of a specified source, use the following format: [source:]?defaultFieldName:newFieldName?, where [source] is the source you want to specify (such as message, trace, or accessLog). Leave newFieldNameblank if you want to omit the field. To rename/omit multiple fields, specify a comma-separated list of field name mappings.
```

Comment:
suggested edit-

When logs are in JSON format, use this attribute to replace default field names with new field names or to omit fields from the logs. To replace a field name, configure the new field name by using the following format: defaultFieldName:newFieldName?. For field names that are associated with logs of a specified source, use the following format: [source:]?defaultFieldName:newFieldName?, where [source] is the source you want to specify, such as message, trace, or accessLog. To omit a field from the logs, specify the field name without a replacement, as shown in the following example: defaultFieldName:. To rename or omit multiple fields, specify a comma-separated list of field name mappings.